index.html

```diff
@@ -1,18 +1,147 @@
-<script>
-let name = document.location.pathname.slice(1);
-
-if (!name) {
-  console.log("no place name specfied");
-} else {
-  console.log("fetching place named '" + name + "'");
-
-  fetch(name + '.json')
-    .then(function (response) {
-      return response.text();
-    })
-    .then(function (myText) {
-      console.log(myText);
-    });
-}
-
-</script>
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Yelpington</title>
+    <link rel= 'stylesheet' href="style.css"/>
+		<link
+			rel="stylesheet"
+			href="https://unpkg.com/leaflet@1.3.1/dist/leaflet.css"
+		/>
+    <script src="https://unpkg.com/leaflet@1.3.1/dist/leaflet.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/marked/marked.min.js"></script>
+	</head>
+
+	<body>
+    <div id = 'title'>Yelpington</div>
+		<div id="map"></div>
+		<header id="restaraunt">
+			<ul>
+				Restaraunt List
+				<li>
+					<a href="http://localhost:8080/restaurant.html#ahli-babas-kabob"
```

Comment:
you should be able to link to `/restaurant.html#ahli-babas-kabob` and it will do the same thing. This holds true for the rest of your links as well. You shouldn't need the `http://localhost:8080` since the window already knows where it is.